test/test-utils/sw-test-cleanup.js

```diff
@@ -0,0 +1,22 @@
+window.__testCleanup = () => {
+  const unregisterSW = () => {
```

Comment:
This code would likely be more readable with `async/await`.

`(await window.caches.keys()).map(name => window.caches.delete(name))` etc.